app/views/providers/bank_statements/show.html.erb

```diff
@@ -37,6 +37,14 @@
           <%= t('.info') %>
         </p>
 
+      <div class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+          <span class="govuk-warning-text__assistive"><%= t('generic.warning_assistive') %></span>
+          <%= t('.warning') %>
+        </strong>
+      </div>
```

Comment:
How do you feel about using the [govuk-component] instead here?

It would be something like this:

```suggestion
      <%= govuk_warning_text(text: t(".warning") %>
```

[govuk-component]: https://govuk-components.netlify.app/components/warning-text/#warning-text-with-a-text-argument